gn3/api/datasets.py

```diff
@@ -3,21 +3,19 @@
 from flask import jsonify
 
 from gn3.computations.datasets import create_dataset
+from gn3.computations.datasets import get_traits_data
+from gn3.computations.datasets import get_probeset_trait_data
 
-
-dataset = Blueprint("dataset", __name__)
+from gn3.db_utils import database_connector
```

Comment:
**nitpick:** utility modules can IMHO be a code-smell because it encourages the developer to dump whatever function they want there. General rule of thumb is to keep them as slim as possible.